src/app/services/visit.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+
+import { Visit } from '../models/visit';
+import { environment } from 'src/environments/environment.dev';
```

Comment:
change import path from absolute to relative please

Reply:
the environment path ?

Reply:
`import { environment } from '../../environments/environment.dev';` this?